Source/Laser/LaserParticleContainer.cpp

```diff
@@ -52,6 +52,7 @@ LaserParticleContainer::LaserParticleContainer (AmrCore* amr_core, int ispecies,
         pp.get("wavelength", wavelength);
         pp.get("e_max", e_max);
         pp.query("do_continuous_injection", do_continuous_injection);
+        pp.query("min_particles_per_mode", min_particles_per_mode);
```

Comment:
Could you please add a doc entry for this parameter (and other RZ parameters, if needed)?

Reply:
Done